lib/Config.php

```diff
@@ -36,25 +35,6 @@ public function __construct($configDir) {
                 $this->readData($config);
             }
         }
-
-        $this->debug($this->getValue('info', 'debug', false));
-    }
-
-    public function debug($set = null) {
-        if (is_null($set)) {
-            return $this->debugMode;
-        }
-        elseif ($set) {
-            ini_set('display_errors', 1);
-            ini_set('error_reporting', E_ALL);
-            $this->debugMode = true;
-        }
-        else
-        {
-            ini_set('display_errors', 0);
-            ini_set('error_reporting', E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED);
-            $this->debugMode = false;
-        }
```

Comment:
I was in two minds about this, but I'll have a look at implementing those two INI prefs with the logger